src/Sparrow.Server/Platform/Posix/SmapsReader.cs

```diff
@@ -292,7 +292,13 @@ private int ReadFromFile(Stream fileStream, int bufferIndex)
                             if (foundValue == false)
                                 ThrowNotContainsValidValue(term, currentProcess.Id);
                             if (foundK == false)
-                                ThrowNotContainsKbValue(term, currentProcess.Id);
+                            {
+                                var additionalInfo =
+                                    $"Additional Info: switchBuffer={switchBuffer}, foundK/B={foundK}/{foundValue}, valueSearchPosition={valueSearchPosition}, bytesSearched={bytesSearched}" +
+                                    $", posInTempBuf={posInTempBuf}, searchedBuffer={searchedBuffer}, _tempBufferBytes=<{Encoding.UTF8.GetString(_tempBufferBytes)}" +
+                                    $", buffer 0=<{Encoding.UTF8.GetString(_smapsBuffer[0])}>. buffer 1={Encoding.UTF8.GetString(_smapsBuffer[1])}End of Addtional Info.";
```

Comment:
`Encoding.UTF8.GetString(_smapsBuffer[0])}`  - This is going to get the *whole* buffer, but we didn't read all of it, did we?

Reply:
true, but I want to _see_ the entire buffer. I have the positions where we read and from the text we can understand what went wrong.